packages/ui/src/common/components/typography/TokenValue.tsx

```diff
@@ -15,7 +17,7 @@ interface ValueProps extends ValueSizingProps {
 }
 
 export const TokenValue = React.memo(({ className, value, size }: ValueProps) => {
-  if (!value) {
+  if (value === null || !isDefined(value)) {
```

Comment:
Oh that was unexpected :D But I see that we had `number` there...

Reply:
What do you mean?

Reply:
Just a comment that this bug was a hidden one :) So when `0` was passed instead of `new BN(0)` the value was `-`.

Anyway, closing this.